src/sas/sasgui/perspectives/fitting/fitpage.py

```diff
@@ -212,9 +215,11 @@ def _fill_range_sizer(self):
               "Please enter only the value of interest to customize smearing..."
         smear_message_new_psmear = \
               "Please enter a fixed percentage to be applied to all Q values..."
-        smear_message_2d_x_title = "<dQp>[1/A]:"
-        smear_message_2d_y_title = "<dQs>[1/A]:"
-        smear_message_pinhole_percent_title = "dQ[%]:"
+        smear_message_2d_x_title = "<dQ/Q>p[%]:"
```

Comment:
What do `p` and `s` mean? Maybe there should be a tool tip to help users figure out what is really meant?

Reply:
Not sure why these were labelled this way - agree it makes no sense.. doubly since dqs is said to be "perpendicular."  Since 2D dq is in polar co-ordinates have replaced both the title and tool tip with dq_r and dq_phi.  Tooltip indicates this is polar coordinates. Does this resolve the issue?